downstream/modules/platform/con-controller-resources.adoc

```diff
@@ -0,0 +1,11 @@
+[id="con-controller-resources"]
+
+= Resources Menu
+
+The *Resources* menu provides access to the following components of {ControllerName}.
```

Comment:
```suggestion
The *Resources* menu provides access to the following components of {ControllerName}:
```
Suggesting the colon because the following modules use a colon at the end of the intro sentence.